### PR TITLE
Add Flat File Metadata code doc, Update design guide & research docs, #502, #503

### PR DESCRIPTION
**Pull request by reitermb**
### This pull request changes...

Adds / updates code docs in Design/User Research folders

### CODE REVIEW

File: docs/User-Research/Overview of Research Activities.md [hunk 1]
```diff
@@ -180,7 +179,7 @@ Internally, we also wanted to:
 
 **How we used this research**
 
-- We updated [our stakeholder map](https://github.com/HHS/TANF-app/wiki/Product-Strategy#project-stakeholders) and [user personas](https://github.com/HHS/TANF-app/wiki/User-types).
```
Comment (lfrohlich) [inline docs/User-Research/Overview of Research Activities.md:183]: Could you update these wiki links to be docs links?

Comment (shubhi-raft) [reply docs/User-Research/Overview of Research Activities.md:183]: @reitermb ^^ . @lfrohlich The one you tagged could be Miles doppelganger!

Comment (reitermb) [reply docs/User-Research/Overview of Research Activities.md:183]: @lfrohlich Fixed! Found one more in there too

Comment (lfrohlich) [reply docs/User-Research/Overview of Research Activities.md:183]: oh no! thanks @reitermb

**Review by lfrohlich — APPROVED** [review]
Looks great, thanks!

**Pull request merged**